src/main/java/com/kodarit/stars/toolbox/controller/ImageResizerController.java

```diff
@@ -0,0 +1,75 @@
+package com.kodarit.stars.toolbox.controller;
+
+import com.kodarit.stars.toolbox.ImageResizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.IOException;
+
+@Controller
+public class ImageResizerController {
+
+    private static final Logger logger = LoggerFactory.getLogger(ImageResizerController.class);
+    private final ImageResizer imageResizer;
+    private static final long MAX_FILE_SIZE = 10 * 1024 * 1024; // 10MB limit
+
+    @Autowired
+    public ImageResizerController(ImageResizer imageResizer) {
+        this.imageResizer = imageResizer;
+    }
+
+    @GetMapping("/tools/image-resizer")
+    public String showImageResizerPage() {
+        return "image-resizer";
+    }
+
+    @PostMapping("/tools/image-resizer/process")
```

Comment:
If you're going to name this like this, previous tools should probably also be updated